### DISCUSSION TITLE
favicon

### DISCUSSION THREAD

**Discussion by bluefander** [kdevo/osprey-delight#38] [Q&A] [answered]
Hi

I am experiencing a strange behaviour with the favicon I specified in my config.yaml.
When I am running "hugo server -D" I can see the icon I specified there, but when I am outputting my hugo site with "hugo -D" I see the default osprey delight favicon. Is this reproducable?

Thanks!

**Comment by kdevo** [comment, marked as answer]
Is the correct favicon inside your `public` directory? If yes, this is likely a browser caching issue.

**Reply by bluefander** [reply]
Oh, it probably was. Wasn't aware that my browser cache could cause this.
Thanks!
